openedx/core/lib/xblock_utils.py

```diff
@@ -211,6 +211,23 @@ def grade_histogram(module_id):
     return grades
 
 
+def add_grading_markup(course, block, view, frag, context):  # pylint: disable=unused-argument
```

Comment:
> Please add docstring.

Successfully added. I don't know the types of functions arguments. Is it enough? 
https://github.com/raccoongang/edx-platform/blob/anton/grading-v2/openedx/core/lib/xblock_utils.py#L215